.github/workflows/autobuild.yml

```
@@ -212,12 +211,18 @@ jobs:
           key:                      ${{ matrix.config.target_os }}-${{ hashFiles('.github/workflows/autobuild.yml', '.github/autobuild/android.sh') }}-${{ matrix.config.base_command }}
 
       - name:                       Set up build dependencies for ${{ matrix.config.config_name }}
+        id:                         setup
         run:                        ${{ matrix.config.base_command }} setup
         env:
           JAMULUS_BUILD_VERSION:    ${{ needs.create_release.outputs.build_version }}
+          MACOS_CERTIFICATE:        ${{ secrets.MACOS_CERT}}
```

If we need to keep this at all, the missing space should be added

```suggestion
          MACOS_CERTIFICATE:        ${{ secrets.MACOS_CERT }}
```

---

I just copied and pasted from the Build step. I see the space is missing there too, but I didn't notice before.